tfjs-backend-webgpu/src/kernels/FromPixelsExternalImage.ts

```diff
@@ -88,6 +88,9 @@ export function fromPixelsExternalImage(args: {
 
   backend.recordFromPixelsCommands(
       program, info.bufferInfo.buffer, layout, externalResource);
-  backend.submitQueue();
+  if (env().get('WEBGPU_DEFERRED_SUBMIT_BATCH_SIZE') as
+      number <= backend.computePassNumberInEncoder) {
+    backend.submitQueue();
+  }
```

Comment:
Could you please also move above `if` block to `recordFromPixelsCommands`? Maybe rename `recordFromPixelsCommands` to `runFromPixelsProgram`.